geomesa-spark/geomesa-spark-jts/src/test/scala/org/locationtech/geomesa/spark/SparkSQLSpatialRelationshipsTest.scala

```diff
@@ -0,0 +1,505 @@
+/***********************************************************************
+ * Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.spark
+
+import java.{lang => jl}
+
+import com.typesafe.scalalogging.LazyLogging
+import com.vividsolutions.jts.geom.Point
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.jts.JTSTypes
+import org.apache.spark.sql.{Column, TypedColumn, _}
+import org.junit.runner.RunWith
+import org.locationtech.geomesa.spark.SQLGeometricConstructorFunctions._
+import org.locationtech.geomesa.spark.SQLSpatialFunctions._
+import org.locationtech.geomesa.spark.SparkDefaultEncoders._
+import org.locationtech.geomesa.spark.SpatialEncoders._
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class SparkSQLSpatialRelationshipsTest extends Specification with BlankDataFrame with LazyLogging {
```

Comment:
Are you still doing any logging here?

Reply:
Looks like cruft from when I copied the test guts from `geomesa-spark-sql`. I'll remove.